tests/utils.go

```diff
@@ -251,7 +251,7 @@ func RunVMIAndExpectSchedulingWithWarningPolicy(vmi *v1.VirtualMachineInstance,
 	)
 }
 
-func getRunningPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace string) (*k8sv1.Pod, error) {
+func GetRunningPodByVirtualMachineInstanceOrError(vmi *v1.VirtualMachineInstance, namespace string) (*k8sv1.Pod, error) {
```

Comment:
Ack, thanks.

Reply:
`GetRunningPodByVirtualMachineInstanceOrError` is used by `GetRunningPodByVirtualMachineInstance` which is used by many other tests. I don't think it can be inlined into `libvirtDomainIsPaused` because of it.

Reply:
I'm suggesting to leave `getRunningPodByVirtualMachineInstance` as it is. Just take the 4 lines you need from it, and put them in `libvirtDomainIsPaused`.

Reply:
Done.